tests/README.md

```
@@ -0,0 +1,238 @@
+# Test Architecture
+
+This document describes the organization and usage of the `climate_indices` test suite.
+
+## Directory Structure
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)